src/guide/plugins.md

```diff
@@ -63,3 +91,36 @@ MyPlugin.install = (app, options) => {
   })
 }
 ```
+
+## Using a Plugin
+
+After a Vue app has been initialized with `createApp()`, you can add a plugin to your application by calling the `use()` method. 
+
+We will use the `i18nPlugin` we created in the [Writing a Plugin](#writing-a-plugin) section for demo purposes.
+
+The `use()` method takes two parameters. The first one is the plugin to be installed, in this case `i18nPlugin`. 
+
+The second parameter is optional, and depends on each particular plugin. In the case of the demo `i18nPlugin`, it is an object with the translated strings.
+
+:::tip
+If you are using third party plugins such as `Vuex` or `Vue Router`, always check the documentation to know what that particular plugin expects to receive as a second parameter.
+:::
+
+```js
+import { createApp } from 'vue'
+import App from './App.vue'
+
+const app = createApp(App)
+const i18nStrings = {
+  greetings: {
+    hi: 'Hallo!'
+  }
+}
+
+app.use(i18nPlugin, i18nStrings)
+app.mount('#app')
+```
+
+`app.use()` automatically prevents you from using the same plugin more than once, so calling it multiple times on the same plugin will install the plugin only once.
```

Comment:
```suggestion
```

This could probably go higher in this section since it's a generic piece of advice prior to getting into specifics.